desktop/apps/artwork/components/partner_stub/index.jade

```diff
@@ -28,19 +28,9 @@ unless artwork.is_in_auction
       if artwork.is_inquireable && contacts && contacts.length
         .artwork-partner-stub__phone
           a(
-            class='js-artwork-partner-stub-phone-toggle analytics-artwork-show-phone-number'
+            class='js-artwork-partner-stub-inquire-via-phone analytics-artwork-show-phone-number'
             data-partner_id= partner._id
             data-artwork_id= artwork._id
+            data-partner= partner
```

Comment:
Apparently `partner` gets serialized into json and you can get that value by `e.toElement.dataset.partner`(which you can see above), but I could be missing something.